core/src/main/kotlin/org/evomaster/core/problem/rest/service/RestResourceFitness.kt

```diff
@@ -206,6 +206,7 @@ class RestResourceFitness : AbstractRestFitness<RestIndividual>() {
         if (config.extractSqlExecutionInfo && config.probOfEnablingResourceDependencyHeuristics > 0.0)
             dm.updateResourceTables(individual, dto)
 
+        // FIXME: Inclusion of HostnameResolutionAction will have any impact?
```

Comment:
this will be refactored... so for now it is fine to just leave a `FIXME` comment